pvacseq/lib/convert_vcf.py

```diff
@@ -105,6 +155,38 @@ def main(args_input = sys.argv[1:]):
             transcript_expns[row['tracking_id']] = row
         args.transcript_expn_file.close()
 
+    coverage = {}
+    if args.normal_snvs_coverage_file is not None:
+        if 'snvs' not in coverage:
+            coverage['snvs'] = {}
+        coverage['snvs']['normal'] = parse_bam_readcount_file(args.normal_snvs_coverage_file)
+        args.normal_snvs_coverage_file.close()
+    if args.normal_indels_coverage_file is not None:
+        if 'indels' not in coverage:
+            coverage['indels'] = {}
+        coverage['indels']['normal'] = parse_bam_readcount_file(args.normal_indels_coverage_file)
+        args.normal_indels_coverage_file.close()
+    if args.tdna_snvs_coverage_file is not None:
+        if 'snvs' not in coverage:
+            coverage['snvs'] = {}
+        coverage['snvs']['tdna'] = parse_bam_readcount_file(args.tdna_snvs_coverage_file)
+        args.tdna_snvs_coverage_file.close()
+    if args.tdna_indels_coverage_file is not None:
+        if 'indels' not in coverage:
+            coverage['indels'] = {}
+        coverage['indels']['tdna'] = parse_bam_readcount_file(args.tdna_indels_coverage_file)
+        args.tdna_indels_coverage_file.close()
+    if args.trna_snvs_coverage_file is not None:
+        if 'snvs' not in coverage:
+            coverage['snvs'] = {}
+        coverage['snvs']['trna'] = parse_bam_readcount_file(args.trna_snvs_coverage_file)
+        args.trna_snvs_coverage_file.close()
+    if args.trna_indels_coverage_file is not None:
+        if 'indels' not in coverage:
+            coverage['indels'] = {}
+        coverage['indels']['trna'] = parse_bam_readcount_file(args.trna_indels_coverage_file)
+        args.trna_indels_coverage_file.close()
```

Comment:
Would it make sense to have something like:
```python
def add_coverage_info(coverage, file, variant_type, data_type):
  if file is not None:
    if variant_type not in coverage:
      coverage[variant_type] = {}
    coverage[variant_type][data_type] = parse_bam_readcount_file(file)
    file.close()
```
and call it for each of these files?